go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -899,6 +905,65 @@ func (bnc *BaseNetworkController) findMigratablePodIPsForSubnets(subnets []*net.
 	return ipList, nil
 }
 
+func (bnc *BaseNetworkController) AddResourceCommon(objType reflect.Type, obj interface{}) error {
+	switch objType {
+	case factory.PolicyType:
+		np, ok := obj.(*knet.NetworkPolicy)
+		if !ok {
+			return fmt.Errorf("could not cast %T object to *knet.NetworkPolicy", obj)
+		}
+
+		if err := bnc.addNetworkPolicy(np); err != nil {
+			klog.Infof("Network Policy add failed for %s/%s, will try again later: %v",
+				np.Namespace, np.Name, err)
+			return err
+		}
+	default:
+		return fmt.Errorf("no add function for object type %s", objType)
+	}
+	return nil
+}
+
+func (bnc *BaseNetworkController) UpdateResourceCommon(objType reflect.Type, oldObj, newObj interface{}) error {
+	switch objType {
+	case factory.PolicyType:
+		oldNp, ok := oldObj.(*knet.NetworkPolicy)
+		if !ok {
+			return fmt.Errorf("could not cast obj of type %T to *knet.NetworkPolicy", oldObj)
+		}
+		if err := bnc.deleteNetworkPolicy(oldNp); err != nil {
+			klog.Infof("NetworkPolicy delete failed for %s/%s, will try again later: %v",
+				oldNp.Namespace, oldNp.Name, err)
+			return err
+		}
+		newNp, ok := newObj.(*knet.NetworkPolicy)
+		if !ok {
+			return fmt.Errorf("could not cast obj of type %T to *knet.NetworkPolicy", newObj)
+		}
+		if err := bnc.addNetworkPolicy(newNp); err != nil {
+			klog.Infof("NetworkPolicy add failed for %s/%s, will try again later: %v",
+				newNp.Namespace, newNp.Name, err)
+			return err
+		}
+	default:
+		return fmt.Errorf("object type %s not supported", objType)
+	}
+	return nil
+}
```

Comment:
I don't think update for network policies is going to be called, from 
https://github.com/jcaamano/ovn-kubernetes/blob/14afcbf9621264f6d1de32107bfdcf77f655bbbc/go-controller/pkg/ovn/base_event_handler.go#L25-L26

Reply:
right, removed the method as it's no longer needed.